supervised/regularization.md

```diff
@@ -99,7 +102,8 @@ We prove now the bias-variance tradeoff equality. $$\begin{aligned}
 &= \mathbb{E}[\eta^2] + \mathbb{E}[(f - \mathbb{E}\hat{f} + \mathbb{E}\hat{f} - \hat{f})^2]&\\
```

Comment:
The line

```
&= \mathbb{E}[\eta^2]+\mathbb{E}[(f-\hat{f})^2]&[\text{uses } \eta \rotatebox[origin=c]{90}{$\models$}\hat{f}]\\
```

uses `\rotatebox`; I'm not sure if Katex supports this. The entire latex block isn't rendering correctly for me...

Reply:
yep the regularization notes https://gradml.mit.edu/supervised/regularization/ is in less "polished" shape than others with all sorts of rendering/ latex compatibility issues. I'll need to fix them soon.